soc/nordic/common/mram_latency.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * nRF SoC specific public APIs for MRAM latency management
+ */
+
+#ifndef SOC_NORDIC_COMMON_MRAM_LATENCY_H_
+#define SOC_NORDIC_COMMON_MRAM_LATENCY_H_
+
+#include <zephyr/sys/onoff.h>
```

Comment:
I'd try to be consistent with gpd: use some namespacing, place it under soc/nrf54h/..., headers `nrf/mram_latency.h`, etc.

Reply:
I did not put it in nrf54h because of nrf92 and took the similar approach to dmm. I can move it to nrf54h however i see that it is already a bit unclear with regards to include files (only gdp has it's own include path). Maybe we should have `soc/nordic/include/nrf` path?